modules/pipeline/providers/resourcegc/log.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourcegc
+
+var (
+	logPrefix = "[resourcegc]: "
+)
+
+func (p *provider) infof(format string, args ...interface{}) {
```

Comment:
no need this prefix anymore.
`module name` will be added automatically by logrus fields.

Reply:
adjusted